README.md

```diff
@@ -445,6 +448,146 @@ No, see the [cited document](https://eclass.eu/fileadmin/Redaktion/pdf-Dateien/B
+          "securityAttributes": [ { "type": "NONE", "key": "NONE", "value": "NONE" } ]
+      }
+    }
+  ]
+}
+```
+
+A service implementing a complete profile, e.g. "https://admin-shell.io/aas/API/3/0/SubmodelRepositoryServiceSpecification/SSP-002" (only READ API Operations), may want to outline the location of the additionally required APIs (Serialization and Description). Each of the endpoints is therefore stated in an own entry in the endpoints array. In particular for the Serialization API, it is good practice to already include all needed parameters - "submodelIds" or additionally "aasIds" for an AAS Repository - to simplify the call for the clients.
+
+Example for a Submodel provided through a submodel repository in conjunction with the Serialization and Description APIs:
+```
+{ 
+  ...
+  "endpoints": [
+    { 
+      "interface": "SUBMODEL-REPOSITORY-3.0" ,
+      "protocolInformation": { 
+          "href": "https://<hostname>/path-to-submodel/api/v3.0/submodels/c3VibW9kZWwtMTIz"
+          "endpointProtocol": "HTTP",
+          "endpointProtocolVersion: [ "1.1" ],
+          "securityAttributes": [ { "type": "NONE", "key": "NONE", "value": "NONE" } ]
+      }
+    },
+    { 
+      "interface": "DESCRIPTION-3.0" ,
+      "protocolInformation": { 
+          "href": "https://<hostname>/path-to-submodel/api/v3.0/description"
+          "endpointProtocol": "HTTP",
+          "endpointProtocolVersion: "1.1",
+          "securityAttributes": [ { "type": "NONE", "key": "NONE", "value": "NONE" } ]
+      }
+    },
+    { 
+      "interface": "SERIALIZE-3.0" ,
+      "protocolInformation": { 
+          "href": "https://<hostname>/path-to-submodel/api/v3.0/serialization?submodelIds=c3VibW9kZWwtMTIz"
+          "endpointProtocol": "HTTP",
+          "endpointProtocolVersion: [ "1.1" ],
+          "securityAttributes": [ { "type": "NONE", "key": "NONE", "value": "NONE" } ]
+      }
+    }
+  ]
+}
+```
+
+
+In case the endpoints are provided in conjunction with other specifications, e.g., in dataspaces, the remaining fields can be used to provide more information to the client. In particular, the `subprotocol` shall state which combination of AAS with which other specification is used. In case IDS connectors control the access to AAS endpoints, `DSP` (read like "AAS over the Dataspace Protocol") can be used. The `subprotocolBody` can for instance provide information on dataspace-specific data asset identifiers as well as authorisation endpoints.
+
+Example for a Submodel endpoint which is offered through a dataspace connector:
+```
+{
+  ...
+  "endpoints": [
+    { 
+      "interface": "SUBMODEL-3.0",
+      "protocolInformation": {
+          "href": "https://provider-edc.data.plane/some-submodel/api/v3.0/submodel",
+          "endpointProtocol": "HTTP",
+          "endpointProtocolVersion: [ "1.1" ],
+          "subprotocol": "DSP",
+          "subprotocolBody": "id=123;dspEndpoint=http://edc.control.plane/",
```

Comment:
This is correct, as it is defined in Catena-X release right now.
https://github.com/catenax-ng/eclipse-tractusx.github.io/blob/main/docs-kits/kits/Traceability%20Kit/Software%20Development%20View/page_software-development-view.md

It may change in the future - not release yet option is this:
```
"subprotocolBody": "providerDspBasePath=https://edc.control.plane/api/v1/dsp;dataset=123",
```
Reasons: `dataset` is the correct an more specific expression coming from DSP. Also, the `base path` must be included, not only the domain name. Because only blow that base path, the endpoints are standardized as part of the DSP http binding.

Since this is a best practices document, I would leave it as it is right now. Or what is your opinion @sebbader-sap ?

Reply:
Same as yours. It's only an example.